src/renderer/components/+catalog/catalog-menu.tsx

```diff
@@ -42,39 +40,51 @@ function Item(props: TreeItemProps) {
   );
 }
 
-export const CatalogMenu = observer((props: Props) => {
-  return (
-    // Overwrite Material UI styles with injectFirst https://material-ui.com/guides/interoperability/#controlling-priority-4
-    <StylesProvider injectFirst>
-      <div className="flex flex-col w-full">
-        <div className={styles.catalog}>Catalog</div>
-        <TreeView
-          defaultExpanded={["catalog"]}
-          defaultCollapseIcon={<Icon material="expand_more"/>}
-          defaultExpandIcon={<Icon material="chevron_right" />}
-          selected={props.activeItem || "browse"}
+interface Dependencies {
+}
+
+const NonInjectedCatalogMenu = observer(({ categories, onItemClick, activeItem }: Dependencies & CatalogMenuProps) => (
+  // Overwrite Material UI styles with injectFirst https://material-ui.com/guides/interoperability/#controlling-priority-4
+  <StylesProvider injectFirst>
+    <div className="flex flex-col w-full">
+      <div className={styles.catalog}>Catalog</div>
+      <TreeView
+        defaultExpanded={["catalog"]}
+        defaultCollapseIcon={<Icon material="expand_more"/>}
+        defaultExpandIcon={<Icon material="chevron_right" />}
+        selected={activeItem || "browse"}
+      >
+        <Item
+          nodeId="browse"
+          label="Browse"
+          data-testid="*-tab"
+          onClick={() => onItemClick("*")}
+        />
+        <Item
+          nodeId="catalog"
+          label={<div className={styles.parent}>Categories</div>}
+          className={cssNames(styles.bordered)}
         >
-          <Item nodeId="browse" label="Browse" data-testid="*-tab" onClick={() => props.onItemClick("*")}/>
-          <Item
-            nodeId="catalog"
-            label={<div className={styles.parent}>Categories</div>}
-            className={cssNames(styles.bordered)}
-          >
-            {
-              getCategories().map(category => (
-                <Item
-                  icon={getCategoryIcon(category)}
-                  key={category.getId()}
-                  nodeId={category.getId()}
-                  label={category.metadata.name}
-                  data-testid={`${category.getId()}-tab`}
-                  onClick={() => props.onItemClick(category.getId())}
-                />
-              ))
-            }
-          </Item>
-        </TreeView>
-      </div>
-    </StylesProvider>
-  );
+          {
+            categories.get().map(category => (
+              <Item
+                icon={getCategoryIcon(category)}
+                key={category.getId()}
+                nodeId={category.getId()}
+                label={category.metadata.name}
+                data-testid={`${category.getId()}-tab`}
+                onClick={() => onItemClick(category.getId())}
+              />
+            ))
+          }
+        </Item>
+      </TreeView>
+    </div>
+  </StylesProvider>
+));
+
+export const CatalogMenu = withInjectables<Dependencies, CatalogMenuProps>(NonInjectedCatalogMenu, {
```

Comment:
No dependencies?

Reply:
Ah, oops. Past in the one dependency as a prop by accident.